Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -146,10 +147,27 @@ void MolDraw2D::drawMoleculeWithHighlights(
     const map<int, double> &highlight_radii,
     const map<int, int> &highlight_linewidth_multipliers, int confId) {
   setupTextDrawer();
-  MolDraw2D_detail::DrawMol *dm = new MolDraw2D_detail::DrawMolMCH(
-      mol, legend, panelWidth(), panelHeight(), drawOptions(), *text_drawer_,
-      highlight_atom_map, highlight_bond_map, highlight_radii,
-      highlight_linewidth_multipliers, confId);
+  MolDraw2D_detail::DrawMol *dm = nullptr;
+  switch (drawOptions().multiColourHighlightStyle) {
+    case MultiColourHighlightStyle::CIRCLEANDLINE:
+      dm = new MolDraw2D_detail::DrawMolMCHCircleAndLine(
+          mol, legend, panelWidth(), panelHeight(), drawOptions(),
+          *text_drawer_, highlight_atom_map, highlight_bond_map,
+          highlight_radii, highlight_linewidth_multipliers, confId);
+      break;
+    case MultiColourHighlightStyle::LASSO:
+      dm = new MolDraw2D_detail::DrawMolMCHLasso(
+          mol, legend, panelWidth(), panelHeight(), drawOptions(),
+          *text_drawer_, highlight_atom_map, highlight_bond_map,
+          highlight_radii, highlight_linewidth_multipliers, confId);
+      break;
+    default:
+      dm = new MolDraw2D_detail::DrawMolMCHCircleAndLine(
```

Comment:
shouldn't this case be an error?
How is it possible to encounter an unknown multicolour highlight style here?